core/server/types/helmrelease.go

```diff
@@ -25,12 +27,14 @@ func HelmReleaseToProto(helmrelease *v2beta1.HelmRelease, clusterName string, in
 		Interval:    durationToInterval(helmrelease.Spec.Interval),
 		HelmChart: &pb.HelmChart{
 			Chart:     helmrelease.Spec.Chart.Spec.Chart,
-			Namespace: helmrelease.Spec.Chart.Spec.SourceRef.Namespace,
-			Name:      helmrelease.Spec.Chart.Spec.SourceRef.Name,
 			Version:   helmrelease.Spec.Chart.Spec.Version,
+			Name:      fmt.Sprintf("%s-%s", helmrelease.Namespace, helmrelease.Name),
```

Comment:
Why is it `namespace-name` and not just `name`?

Reply:
The way I created the testing helm objects was through `flux create source helm ...` and `flux create helmrelease podinfo....`, after that, I saw that flux automatically created a resource called `flux-system-podinfo`.